sklearn/decomposition/dict_learning.py

```diff
@@ -717,6 +717,19 @@ def dict_learning_online(X, n_components=2, alpha=1, n_iter=100,
         Number of iterations run. Returned only if `return_n_iter` is
         set to `True`.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.decomposition import DictionaryLearning
+    >>> X = np.array([[0., 0., 1.],
+    ...               [1.,0.,0.],
```

Comment:
please avoid flake8 errors in examples

Reply:
The flake8 test didn't pick them up.  I suppose it is because it is in a comment?